src/components/nav.js

```diff
@@ -1,21 +1,34 @@
 import React from "react"
-
+import { slide as Menu } from "react-burger-menu"
+import { Link } from "gatsby"
 import Logo from "../images/logo.svg"
 import * as Styles from "../styles/components/nav"
+import { Z_BUF_ERROR } from "zlib"
+import "../styles/hamburger.css"
```

Comment:
It looks like react-burger-menu supports JS styling, we should use that instead

Reply:
I moved them all into the nav.js for styles